src/@ionic-native/plugins/secure-storage/index.ts

```diff
@@ -125,7 +125,7 @@ export class SecureStorage extends IonicNativePlugin {
       if (checkAvailability('cordova.plugins.SecureStorage', null, 'SecureStorage') === true) {
         const instance = new cordova.plugins.SecureStorage(() => res(new SecureStorageObject(instance)), rej, store);
       } else {
-        res(new SecureStorageObject());
+        rej('SecureStorage failure: cordova plugin is not available');
```

Comment:
Sorry forget about the comment above, here's a better way to do it:
```
@CordovaCheck()
create(): Promise<SecureStorageObject> {
  return new Promise<SecureStorageObject>((resolve: Function) => {
    const instance = new cordova.plugins.SecureStorage(...);
  });
}
```

The `@CordovaCheck()` decorator will make it only run if the `checkAvailability` passes, and it will automatically reject with the error if it exists.